Documentation/ceph-object-bucket-claim-crd.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Object Bucket Claim CRD
```

Comment:
I'd suggest changing the title to "Object Bucket Claim". This topic will talk about much more than just the CRD.

Reply:
Done